_maps/map_files/generic/centcomm.dmm

```diff
@@ -13957,11 +13957,12 @@
 /turf/simulated/floor/plasteel/dark,
 /area/centcom/specops)
 "Xo" = (
-/obj/machinery/door_control/no_emag/no_cyborg{
+/obj/machinery/door_control/no_emag{
```

Comment:
I dont think you meant to remove this
```suggestion
/obj/machinery/door_control/no_emag/no_cyborg{
```

Reply:
Was intentional, that button controls the shutters from the other side, in case a ert borg wishes to close them.